lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -46,26 +46,29 @@ export class EvaluationClientWrapper {
       const credentials: VerifiableCredential[] = matches.map(
         (e) => jp.nodes(this._client.verifiableCredential, e)[0].value
       );
+      const areRequiredCredentialsPresent = this.determineAreRequiredCredentialsPresent(matchSubmissionRequirements);
       selectResults = {
         errors: errors,
         matches: [...matchSubmissionRequirements],
-        verifiableCredentials: [...credentials],
+        areRequiredCredentialsPresent,
+        selectableVerifiableCredentials: [...credentials],
         warnings,
       };
     } else {
       const marked: HandlerCheckResult[] = this._client.results.filter(
         (result) => result.evaluator === 'MarkForSubmissionEvaluation' && result.status !== Status.ERROR
       );
-
       const matchSubmissionRequirements = this.matchWithoutSubmissionRequirements(marked, presentationDefinition);
       const matches = this.extractMatches(matchSubmissionRequirements);
       const credentials: VerifiableCredential[] = matches.map(
         (e) => jp.nodes(this._client.verifiableCredential, e)[0].value
       );
+      const areRequiredCredentialsPresent = this.determineAreRequiredCredentialsPresent(matchSubmissionRequirements);
```

Comment:
After moving above call below the If-else of submissionrequirements presence check to avoid redundancy this line can be removed.